packages/@okta/vuepress-site/docs/release-notes/2022/index.md

```diff
@@ -1,6 +1,74 @@
+
+Admins can configure the Okta OIDC Identity Provider to only authenticate users from an inbound OIDC IdP if their usernames match a predefined regular expression pattern. See the `filter` property from the [Subject Policy object](/docs/reference/api/idps/#subject-policy-object) in the IdPs API and the Authentication Settings section in the [OIDC IdP](/docs/guides/add-an-external-idp/openidconnect/main/#create-an-identity-provider-in-okta) configuration. <!-- OKTA-500903 -->
+
+#### Signed request support for generic SAML IdP is EA in Preview
+
+Using signed SAML requests ensures that incoming requests are from genuine applications. When signed SAML request is configured, Okta only accepts SAML requests signed using the certificate associated with the app integration. Having signed SAML requests also resolves scenarios where the Assertion Consumer Service (ACS) URL requested after authentication can be one of several domains or URLs. When a Service Provider sends a signed authentication request, Okta can accept dynamic ACS values as part of the SAML request and posts the SAML assertion response to the ACS value specified in the request. See the Advanced Settings section of [Create SAML app integrations using AIW](https://help.okta.com/okta_help.htm?type=oie&id=csh-apps-aiw-saml). <!-- OKTA-493043 -->
+
+#### System Log events for telephony rate limit violations
+
+Telephony `system.sms.send.*` and `system.voice.send.*` events are now issued as a `DENY` System Log message when SMS or Voice messages are blocked due to telephony operational rate limit violations. The `system.operation.rate_limit.violation` event is still fired, but will be deprecated in the 2022.08.0 release.
+
+Additionally, the way that the `MobilePhoneID` hash is created for all `system.sms.send.*` and `system.voice.send.*` events is changed.
+
+See [System Log API](/docs/reference/api/system-log/). <!-- OKTA-498664 -->
+
+#### Bugs fixed in 2022.06.0
+
+* When a create user API request failed due to non-compliant user credentials, the "Add user to group membership" and "Add user to application membership" events were listed incorrectly in the System Log. (OKTA-469408)
+
+* No System Log event was triggered when a request to [Forgot password API endpoint](/docs/reference/api/users/#forgot-password) (`/users/{userId}/credentials/forgot_password`) was made for a user with LOCKED_OUT status. (OKTA-4852442)
+
+* No error messages were returned when an API request was made to [create](/docs/reference/api/roles/#create-resource-set) or [update](/docs/reference/api/roles/#add-more-resources) a resource set with invalid ORNs. This occurred if the request was made to an org with the **Okta Resource Name (ORN) in API for Administrator Roles** feature enabled. (OKTA-499775)
+
+* When a user was created with `activate=true` and `nextLogin=changePassword` parameters, their user status was set to ACTIVE instead of PASSWORD_EXPIRED. (OKTA-501729)
```

Comment:
```suggestion
* When a user was created with `activate=true` and `nextLogin=changePassword` parameters, their user status was set to `ACTIVE` instead of `PASSWORD_EXPIRED`. (OKTA-501729)
```